code/__DEFINES/dcs/signals.dm

```diff
@@ -1151,6 +1151,7 @@
 // /datum/component/storage signals
 
 ///() - returns bool.
+
```

Comment:
```suggestion
```
want to remove the lines that was added to this file so its not needlessly updated